### ISSUE TITLE
Time scale for EPOCHs

### ISSUE THREAD

**Issue by paulray** [nanograv/PINT#690]
I noticed that POSEPOCH and DMEPOCH are in the ‘utc’ scale:

``` 
In [98]: m.POSEPOCH.quantity                                                                                                                                      
Out[98]: <Time object: scale='utc' format='pulsar_mjd_string' value=53750.0000000000000000>

 In [107]: m2.DMEPOCH.quantity                                                                                                                                     
Out[107]: <Time object: scale='utc' format='pulsar_mjd_string' value=54890.0000000000000000>
```

Is that correct? Shouldn’t they be in the TDB scale like the other epochs?

 ```
In [104]: m2.PEPOCH.quantity                                                                                                                                      
Out[104]: <Time object: scale='tdb' format='pulsar_mjd_string' value=54890.0000000000000000>

In [106]: m2.T0.quantity                                                                                                                                          
Out[106]: <Time object: scale='tdb' format='pulsar_mjd_string' value=53113.9550955988560900>
```

When DMEPOCH and POSEPOCH are used, it is by subtracting them from the 'tdbld' column of the TOAs table, which is certainly in TDB units, so I think they should be in TDB as well.

**Comment by rossjjennings** [comment]
The epoch-changing functionality that I added also assumes these are TDB. Somehow I never noticed that they were being read in as UTC.

**Comment by paulray** [comment]
Fixed in #697
